platform-includes/configuration/capture-console/javascript.mdx

```diff
@@ -9,7 +9,31 @@ Sentry.init({
 });
 ```
 
-```html {tabTitle: Loader}
+```html {tabTitle: Loader (v7)}
+<script>
+  // Configure sentryOnLoad before adding the Loader Script
+  window.sentryOnLoad = function () {
+    Sentry.init({});
+
+    const integration = Sentry.captureConsoleIntegration?.();
+    if (integration) {
+      Sentry.addIntegration(integration);
+    }
```

Comment:
Optional chaining is not that widely supported yet I believe 

```suggestion
    if (Sentry.captureConsoleIntegration) {
      Sentry.addIntegration(Sentry.captureConsoleIntegration());
    }
```

Reply:
It's a good point, I rewrote all like this!